styx-common/src/test/java/com/spotify/styx/state/RunStateTest.java

```diff
@@ -510,9 +507,7 @@ public void testAccumulatesMessagesWithInfo() throws Exception {
 
     assertThat(
         transitioner.get(WORKFLOW_INSTANCE).data().messages(),
-        contains(
-            Message.info("info message"),
-            Message.warning("warning message")));
+        contains(Message.warning("warning message")));
```

Comment:
These tests might need renaming, and we should also test the new `.data().message()`